apps/settings/js/fxsync.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+define('fxsync', ['modules/settings_utils', 'shared/settings_listener'
+], function(SettingsUtils, SettingsListener) {
+  var FxSync = {
+    init: function fmd_init() {
+      // The app name may overflow the header width in some locales; try to
+      // shrink it. Bug 1087441
+      var header = document.querySelector('#fxsync gaia-header');
+      SettingsUtils.runHeaderFontFit(header);
+    },
+  };
+
+  return FxSync;
+});
+
+navigator.mozL10n.once(function() {
```

Comment:
According to bug 1187668, we don't have to wait `navigator.mozL10n.once` ready anymore.